hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/table/functional/TestHoodieSparkMergeOnReadTableRollback.java

```diff
@@ -218,6 +235,8 @@ void testRollbackWithDeltaAndCompactionCommit(boolean rollbackUsingMarkers, int
       HoodieWriteConfig secondCfg = getHoodieWriteConfigWithSmallFileHandlingOff(true);
       secondCfg.setValue(HoodieTableConfig.VERSION, String.valueOf(tableVersion));
       secondCfg.setValue(HoodieWriteConfig.WRITE_TABLE_VERSION, String.valueOf(tableVersion));
+      secondCfg.setValue(HoodieWriteConfig.TIMELINE_LAYOUT_VERSION_NUM, timelineLayoutVersion);
+
```

Comment:
The timeline layout version is automatically set when setting the write table version in the write config builder, so this change is unnecessary.

Reply:
Do we need to revisit all such cases to make sure the timeline layout version is properly set?